admin-portal/src/pages/pledgeStatus/pledgeStatus.tsx

```diff
@@ -1,43 +1,13 @@
-import React, {useState} from "react";
+import React, {useEffect, useState} from "react";
 import {PageSelection} from "../../types/pages";
 import {Page} from "../../layout/page";
-import {AidPackage, DonorAidPackage} from "../../types/AidPackage";
+import {AidPackage} from "../../types/AidPackage";
 import {Link, useNavigate, useParams} from "react-router-dom";
 import DonorTable from "./donorTable/donorTable";
 import './pledgeStatus.css'
 import ContributionsChart from "../../components/contributionsChart/contributionsChart";
 import {DonorAidPackagePledge} from "../../types/DonarAidPackagePledge";
-
-const demoPackage: DonorAidPackage = {
-  packageId: 0,
-  name: "Aid Package 1",
-  description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam",
-  orderItems: [{
-    orderItemId: 1,
-    medicalItemName: "Paracetamol",
-    quantity: 1000,
-  },
-    {
-      orderItemId: 2,
-      medicalItemName: "Methotrexate",
-      quantity: 100,
-    },
-    {
-      orderItemId: 3,
-      medicalItemName: "Paracetamol",
-      quantity: 500,
-    },
-    {
-      orderItemId: 4,
-      medicalItemName: "Paracetamol",
-      quantity: 500,
-    }
-  ],
-  pledgedPercentage: 40,
-  status: AidPackage.Status.Ordered,
-  supplierID: 0,
-  totalAmount: 2500
-}
```

Comment:
> P.S. My gut feeling says not to keep them so that we could make it more complete. Maybe some kind of mock service would be perfect as you suggested.

Yeah, I agree. We should remove them.
If we want to we'll add a dummy service that's at one place so it doesn't pollute the codebase.